browserify.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var webkitAssign = require('./index');
+
+module.exports = function (file) {
+	return webkitAssign();
+};
```

Comment:
This module seems unnecessary, actually. You ought to be able to just include:

    "transform": [
        "webkit-assign"
    ]

as a browserify transform and call it good.